changelog/26863.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+agent: Correctly constructs kv-v2 secret paths in nested namespaces. The fix is present in the updated consul-template version.
```

Comment:
```suggestion
agent: Correctly constructs kv-v2 secret paths in nested namespaces. 
```

Reply:
Minor suggestion, but I think this sentence could be confusing to users that don't know what consul template is -- and no action is required on their part.